src/lib/pipeline_configuration.py

```diff
@@ -57,11 +57,13 @@ def __init__(self, raw_field, coded_field, coda_filename, cleaner=None, code_sch
         self.id_field = id_field
 
 class PipelineConfiguration(object):  
-    DEV_MODE = True
+    DEV_MODE = False
     
     PROJECT_START_DATE = isoparse("2019-04-02T00:00:00+0300") 
     PROJECT_END_DATE = isoparse("2019-05-14T00:00:00+0300")
 
+    ADVERT_PHONE_NUMBERS_CODE_FILTERS = ["code-NOP-4eb70633", "code-STOP-08b832a8", "code-NA-f93d3eb7","code-NR-5e3eee23"]
```

Comment:
I still think this would be cleaner if you used [Codes.STOP, Codes.TRUE_MISSING, etc.] with a look-up on the code ids happening in the loop which exports the phone numbers, but not a big deal.